packages/utils/src/event-bus/message-aggregator.ts

```diff
@@ -23,6 +26,15 @@ export class MessageAggregator implements IMessageAggregator {
     }
   }
 
+  saveRawMessageData<T>(
+    messageData:
+      | EventBusTypes.MessageFormat<T>
+      | EventBusTypes.MessageFormat<T>[],
+    options?: Record<string, unknown>
+  ): void {
+    this.save(buildEventMessages(messageData, options))
```

Comment:
comment: maybe rename `composeMessage` to `composeMessages` and sligthly change the API, we already have discrepency between the two util